clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -45,7 +45,7 @@ const (
 
 	currentAPICallMetricsVersion = "v1"
 
-	filters = `verb!="WATCH", subresource!="proxy"`
+	filters = `verb!="WATCH", subresource!="exec|proxy"`
```

Comment:
Shouldn't it be subresource!~"exec|proxy"?

Instead it's just a simple string comparison with "exec|proxy" string, not regexp.

Reply:
Definitely It should be

Reply:
Fixed in https://github.com/kubernetes/perf-tests/pull/1814